index.src.html

```diff
@@ -1810,11 +1832,19 @@ <h4 algorithm id="construct-federatedcredential-data">
   granting access to credentials for an origin other than the <a>top-level browsing context</a>,
   which is the only security origin which users can reasonably be expected to understand.
 
-  Therefore, both {{get()}} and {{store()}} will result in a rejected {{Promise}} when called from
-  a [=nested browsing context=], and the API and related interfaces are not exposed inside
-  {{Worker}} contexts.
+  This document exposes the Credential Management APIs to those contexts, as it's likely that some
+  credential types will be straightforward to make available if user agents put enough thought and
```

Comment:
Is "to make available" grammatically correct?

Reply:
It sounds fine to my ear, but I'm happy to reword it to something like "... as it's likely that some credential types will be straightforward to make available _in a cross-origin nested context_ if user agents put enough thought and..." if that would be helpful.

Reply:
@equalsJeffH Do you agree with this proposal? It would be a change to WebAuthN but would not have any impact on the implementations, I guess.

Reply:
The appeal to authority fallacy wins again! :)